packages/gatsby/src/bootstrap/index.js

```diff
@@ -539,3 +509,20 @@ module.exports = async (args: BootstrapArgs) => {
     })
   }
 }
+
+const finishBootstrap = async bootstrapSpan => {
+  report.log(``)
```

Comment:
I would move this under `onPostBootstrap`. The emit is used internally to save our redux state which should happen when bootstrap is completely finished. onPostBootstrap is the last hook that users can fix/update anything they want. For gatsby this is part of the bootstrap process so it should be after from a user perspective it doesn't make much sense as it's 'postBootstrap'.

Does this make sense?